src/components/pages/manage/movies/create.tsx

```diff
@@ -70,22 +130,22 @@ const Create = () => {
 
   useLog({ production: movieState.context.production });
 
+  const steps = useMemo(() => [Step1, Step2, Step3, Step4], []);
+
   return (
     <Page>
       <Page.Title spacing={3} alignItems="center">
         {t(`movies.create.title`)}
       </Page.Title>
       <Page.Content spacing={5}>
-        {[Step1Content, Step2Content, Step3Content, Step4Content].map(
-          (StepContent, index) => (
-            <Step key={index} step={index + 1}>
-              <StepContent
-                movieState={movieState}
-                sendMovieEvent={sendMovieEvent}
-              />
-            </Step>
-          ),
-        )}
+        {steps.map((Step, index) => (
+          <Step
+            movieState={movieState}
+            sendMovieEvent={sendMovieEvent}
+            stepNumber={index + 1}
```

Comment:
Why do we need this `stepNumber` prop if each Step is its own component that has the number in the name? Can't we hardcode the step number of each step in their own component then?

Reply:
Yeah, absolutely. Makes a lot of sense.